astro/projectile.py

```diff
@@ -16,7 +16,7 @@ class Projectile(AstroSprite, Timekeeper):
     """
 
     required_fields = ('imagepath', 'speed', 'damage')
-    defaults = {"angle": 0, 'relative_to_firer_velocity': True}
+    defaults = {"angle": 0, 'relative_to_firer_velocity': True, "fuel_duration":None}
```

Comment:
Use single quotes here and put a space after the colon.

Reply:
There